ArjunaJTA/jta/pom.xml

```diff
@@ -248,65 +251,75 @@
       </activation>
       <dependencies>
         <dependency>
-          <groupId>org.wildfly.arquillian</groupId>
-          <artifactId>wildfly-arquillian-container-managed</artifactId>
+          <groupId>org.jboss.arquillian.core</groupId>
+          <artifactId>arquillian-core-api</artifactId>
           <scope>test</scope>
         </dependency>
         <dependency>
-          <groupId>org.jboss</groupId>
-          <artifactId>jboss-transaction-spi</artifactId>
-          <scope>provided</scope>
-          <exclusions>
-            <exclusion>
-              <groupId>org.jboss.logging</groupId>
-              <artifactId>jboss-logging-spi</artifactId>
-            </exclusion>
-          </exclusions>
+          <groupId>org.jboss.arquillian.protocol</groupId>
+          <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
+          <scope>test</scope>
         </dependency>
         <dependency>
-          <groupId>org.jboss.logging</groupId>
-          <artifactId>jboss-logging</artifactId>
+          <groupId>org.jboss.arquillian.junit</groupId>
+          <artifactId>arquillian-junit-container</artifactId>
           <scope>provided</scope>
         </dependency>
-
         <dependency>
-          <groupId>junit</groupId>
-          <artifactId>junit</artifactId>
+          <groupId>org.jboss.arquillian.extension</groupId>
+          <artifactId>arquillian-extension-byteman</artifactId>
           <scope>test</scope>
         </dependency>
         <dependency>
-          <groupId>org.postgresql</groupId>
-          <artifactId>postgresql</artifactId>
+          <groupId>org.wildfly.extras.creaper</groupId>
```

Comment:
What do we use this for, if it is for managing WildFly how does it differ from using the more standard wildfly maven plugin?